CSC8503/InventoryBuffSystem/PlayerBuffs.cpp

```diff
@@ -64,6 +60,17 @@ void PlayerBuffs::Update(float dt)
 	}
 }
 
+void InventoryBuffSystem::PlayerBuffs::UpdateInventoryObserver(InventoryEvent invEvent, int playerNo)
+{
+	switch (invEvent)
+	{
```

Comment:
The curly braces should be on the previous line. I know coding standards like this are annoying, but please adhere to them. It makes our code more consistent.